txpool/tx_object.go

```diff
@@ -64,14 +64,29 @@ func (o *txObject) Payer() *thor.Address {
 }
 
 func (o *txObject) Executable(chain *chain.Chain, state *state.State, headBlock *block.Header, forkConfig *thor.ForkConfig) (bool, error) {
+	// evaluate the tx on the next block as head block is already history
+	blockNum := headBlock.Number() + 1
+	blockTime := headBlock.Timestamp() + thor.BlockInterval
```

Comment:
Minor suggestion: change from `blockNum` to `nextBlkNum` and `blockTime` to `nextBlkTime`